contracts/ModuleRegistry.sol

```diff
@@ -129,10 +144,11 @@ contract ModuleRegistry is IModuleRegistry, EternalStorage {
      * @param _moduleFactory is the address of the relevant module factory
      * @param _isUpgrade whether or not the function is being called as a result of an upgrade
      */
-    function useModule(address _moduleFactory, bool _isUpgrade) external {
-        if (IFeatureRegistry(getAddressValue(FEATURE_REGISTRY)).getFeatureStatus("customModulesAllowed")) {
+    function useModule(address _moduleFactory, bool _isUpgrade) external nonReentrant {
```

Comment:
useModule() does not need to be guarded for reentrancy. There is no harm in guarding it either so it's fine anyway.

Reply:
`isCompatibleModule` called from `useModule` makes external calls

Reply:
yes but there is no point in re-entering `useModule()`. I don't see any reason why any (malicious) user might want to renter `useModule()`.